lib/assessment.rb

```diff
@@ -5,35 +5,61 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
-##
-# your code here
-##
+class Dog
+
+    attr_reader :name
+    attr_reader :breed
+    attr_reader :age
+
+    def initialize(name, breed, age)
+        @name = name
+        @breed = breed
+        @age = age
+    end
+end
```

Comment:
Yep. Pro-tip:  All the `attr_...` methods can accept multiple symbols as arguments, so you can actually make all your `attr_reader` declarations on a single line.